Corona-Warn-App/src/main/java/de/rki/coronawarnapp/deadman/DeadmanNotificationScheduler.kt

```diff
@@ -0,0 +1,60 @@
+package de.rki.coronawarnapp.deadman
+
+import androidx.work.ExistingPeriodicWorkPolicy
+import androidx.work.ExistingWorkPolicy
+import androidx.work.WorkManager
+import dagger.Lazy
+import dagger.Reusable
+import javax.inject.Inject
+
+@Reusable
+class DeadmanNotificationScheduler @Inject constructor(
+    val timeCalculation: DeadmanNotificationTimeCalculation,
+    val workManager: Lazy<WorkManager>,
```

Comment:
Why is this `Lazy`?

Reply:
Otherwise it wouldn't be injected in main class (CoronaWarnApplication).

Reply:
I think it should work, what error do you get?

Reply:
`java.lang.RuntimeException: Unable to create application de.rki.coronawarnapp.CoronaWarnApplication: java.lang.IllegalStateException: WorkManager is not initialized properly.  You have explicitly disabled WorkManagerInitializer in your manifest, have not manually called WorkManager#initialize at this point, and your Application does not implement Configuration.Provider.`
DeadmanScheduler is being injected along with WorkManager, but it also needs an initialised WorkManager. WorkManager is being initialised right in OnCreate in CoronaWarnApplication (line 61), but after scheduler is being injected.